cerberus/kubernetes/client.py

```diff
@@ -109,3 +109,30 @@ def monitor_component(iteration, component_namespace):
     logging.info("Iteration %s: %s: %s"
                  % (iteration, component_namespace, watch_component_status))
     return watch_component_status, failed_component_pods
+
+
+def monitor_cluster_operator(iteration, cluster_operators):
```

Comment:
Please execute `tox` to ensure the code follows the best practices ( with respect to spacing, blank lines etc. )